src/geometrytools.cpp

```diff
@@ -302,7 +302,7 @@ cv::Mat ransacP3P(const std::vector<cv::Point3f> &scenePoints,
     countit++;
   }
 
-  if (refine == true & (BestInNb >= 4)) {
+  if (refine == true && BestInNb >= 4) {
```

Comment:
What is the behavior difference? Noob question but I always wonder. Is it still doing what we want?

Reply:
They are different in the precedence level: & has higher precedence than && and one of the static analysis tools complained about that. The modification is identical to the original version with brackets wrapped around second expression.